.github/workflows/create-release.yml

```diff
@@ -0,0 +1,81 @@
+name: Create github release & release notes
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+      - master
+    types: [ labeled ]
+  workflow_dispatch:
+
+jobs:
+  save-version:
+    if: github.event.label.name == 'create release'
```

Comment:
Can you do a manual run (workflow_dispatch) with an event label? 🤔 If not, maybe we should remove the workflow_dispatch config above.